anonymizer/base.py

```diff
@@ -84,6 +84,9 @@ def varchar(self, field=None):
         assert field is not None, "The field parameter must be passed to the 'varchar' method."
         max_length = field.max_length
 
+        if not max_length:
+            return ''
```

Comment:
This doesn't seem right `integer` doesn't return 0 if you give it no range . How about a default max length of 50?